src/CMakeLists.txt

```diff
@@ -877,8 +877,14 @@ if(WITH_RADOSGW)
 
       include(BuildArrow)
       build_arrow()
-    endif()
-  endif()
+    endif(WITH_SYSTEM_ARROW)
+
+    if(WITH_RADOSGW_ARROW_FLIGHT)
+      find_package(gRPC)
+      find_package(Protobuf)
+      find_package(c-ares 1.13.0 QUIET)
```

Comment:
so they are optional?

Reply:
Fixed on the build PR. This one is intended to be on top of it, but includes it for the time being.